docs/web/docs/1-getting_started/2-install_and_configure.mdx

```diff
@@ -77,10 +102,125 @@ iambic setup
+<img src={"/img/git/app-icons/icon-dark.png"} style={{width: '25%', display: 'auto', marginLeft: 'auto', marginRight: 'auto'}} />
+
+Download them to your local computer.
+
+1. From the previous section, click on "App Settings".
+<img src={"/img/git/app-icons/step-1-app-settings.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+1. Scroll down to "Display Information", click on "Upload a logo..."
+<img src={"/img/git/app-icons/step-2-display.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+1. Choose either the light icon or dark icon from your computer
+1. Click on the "Set new avatar" button
+<img src={"/img/git/app-icons/step-3-set-avatar.png"} style={{width: '100%', display: 'block', marginLeft: 'auto', marginRight: 'auto'}} />
+
+</details>
+
+### Open a pull request on your **iambic-templates** repo
+
+1. `default_branch=$(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')`
+2. `git checkout -b task/change_description "$default_branch"`
+3. Edit one of your description within your templates, like `resources/github/iam_role_lambda.yaml`. <!-- #TODO edit to what?? -->
+4. Add your changes to your branch, and push it to github:
+	```bash
+	git add resources/github/iam_role_lambda.yaml
+	git commit -m "Change Description"
+	git push origin HEAD
+	```
+5. Create a pull request.
+6. The installed GitHub App will comment with a plan for the changes. If you would like IAMbic to re-plan the changes, you can comment `iambic plan`.
+7. The plan will be provided as an inline comment if it is less than 65kb. Larger plans are uploaded to the `iambic-templates-gist` repository and linked as a comment in the PR
+8. Add reviewers to review your PR and IAMbic's plan in the PR comments.
+9. Once your reviewer approves your changes, add `iambic apply` as a comment to instruct IAMbic to apply the changes. After the Cloud changes are successfully applied, IAMbic will merge the PR into your default branch (Typically main or master).
+
+
+### Configure your GitHub Repository to require approvals before merging
+
+1. Go to the GitHub repository you had created previously, `ExampleOrg/iambic-templates`
+2. Navigate to Repository Settings, in the left navigation menu to "Branches" settings page; the URL will look something like `github.com/ExampleOrg/iambic-templates/settings/branches`
+
+![IAMbic Git Flow](/img/git/github-settings-menu.png)
+
+3. Add a branch protection rule.
+4. Set the branch name pattern to `main` (Or the name of your default branch, if it isn't `main`).
+5. Configure the options
+	1. Enable "Require a pull request before merging".
+	1. Enable "Require Approvals".
+	1. Leave the default value for "Required number of approvals".
+	1. Enable "Require status checks to pass before merging". <!-- #TODO Is there a reason the prior 3 settings were on one line and this was on it's own line?  -->
```

Comment:
I think that's due to logical thinking to separate review approval away from status checks. (may not necessarily to separate them for users)

Reply:
carry over from one of Noah's comments I believe. Just removed it.